cypress/support/elements/tile/TileNavigator.js

```diff
@@ -0,0 +1,25 @@
+class TileNavigator {
+  getTileNavigator() {
+    return cy.get('.primary-workspace [data-testid=tile-navigator]');
+  }
+  getTileNavigatorContainer() {
+    return cy.get('.primary-workspace [data-testid=tile-navigator-container]');
+  }
+  getTileNavigatorPlacementButton() {
+    return cy.get('.primary-workspace [data-testid=tile-navigator-placement-button]');
+  }
+  getTileNavigatorToolbarButton() {
+    return cy.get('.primary-workspace .drawing-toolbar .toolbar-button.navigator');
```

Comment:
You could just use `clueCanvas.clickToolbarButton` and related methods instead of defining these special-purpose selectors.